packages/client/src/arpa_reporter/components/PageNavigation.vue

```diff
@@ -70,98 +70,99 @@
         </ul>
       </div>
     </nav>
-
-    <ul
-      v-if="loggedIn"
-      class="row nav nav-tabs mb-2"
-    >
-      <li class="nav-item">
-        <router-link
-          :class="navLinkClass('/')"
-          to="/"
-        >
-          Dashboard
-        </router-link>
-      </li>
-
-      <li class="nav-item">
-        <router-link
-          :class="navLinkClass('/uploads')"
-          to="/uploads"
-        >
-          Uploads
-        </router-link>
-      </li>
-
-      <li
-        v-if="role === 'admin'"
-        class="nav-item"
+    <main>
```

Comment:
Note to reviewer:  Check hide whitespace to review - only change was to add <main></main> tag.